src/aggregate.h

```diff
@@ -279,11 +278,11 @@ class ClassDeclaration : public AggregateDeclaration
     TypeInfoClassDeclaration *vclassinfo;       // the ClassInfo object for this ClassDeclaration
     bool com;                           // true if this is a COM class (meaning it derives from IUnknown)
     bool cpp;                           // true if this is a C++ interface
+    bool objc;                          // true if this is an Objective-C class/interface
```

Comment:
Classes can only be `com`, `cpp`, or `objc`, but never two at once, right?  These bool flags could be squashed into an enum later, perhaps.